config/sara/sara.yml

```diff
@@ -16,6 +16,9 @@ isolation_non_reporting_max_days: <%= ENV["SARA_ALERT_ISOLATION_NON_REPORING_MAX
 # How many minutes in a reporting period (hint: 1440 mins for 24 hours)
 reporting_period_minutes: <%= ENV["SARA_ALERT_REPORTING_PERIOD_MINUTES"] || 1440 %>
 
+# The number of minutes beyond which a monitoree will be considered non-reporting (hint: 1440 mins for 24 hours)
```

Comment:
Suggest adding to this definition that it's specifically the amount of time since the monitoree's last symptom report.

Reply:
Let me know if this is better! [359af831f042ee77f8a4ec73aa63a2354f7c1a34](https://github.com/SaraAlert/SaraAlert/pull/1208/commits/359af831f042ee77f8a4ec73aa63a2354f7c1a34)